BtmsGateway.Test/BtmsGateway.Test.csproj

```diff
@@ -74,6 +68,15 @@
     <None Update="EndToEnd\Fixtures\TargetRoutingConfig.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Services\Converter\Fixtures\ClearanceRequest.json">
```

Comment:
It might be worth changing this in the same way we did in BTMS - so rather than every fixture file resulting in a change in here, we use a wildcard:

<ItemGroup>
    <Content Include="Scenarios\Samples\**">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>
    
  </ItemGroup>